### DISCUSSION TITLE
Original BittBoy

### DISCUSSION THREAD

**Discussion by lucianogardim** [OnionUI/Onion#541] [Q&A]
Is there any chance to port to the original BittBoy?  It's a Miyoo device too.

**Comment by Olywa** [comment]
Not gonna happen I'm afraid. 

The Bittboy had only a 700mhz cpu (single core) and 32mb RAM. 
By comparison the Mini has a 1.2ghz cpu (dual core) and 128mb ram so is a significantly more powerful device. 

OnionOS is dependant on the Miyoo provided firmware (which the Bittboy does not have). 
A port of Onion would not run well at all and to try to port to an out of production device that few people are still using would not be a good use of the Devs (limited) time.  

Buy yourself a Mini - you won't regret it.
